.github/workflows/pr.yml

```diff
@@ -57,6 +61,9 @@ jobs:
       - name: Setup
         run: source ./scripts/setup.sh
 
+      - name: Check formatting
+        run: python3 ./scripts/check-format.py

```

Comment:
why not a pwsh script for consistency?  (L34 also)

Reply:
I was considering a proposal Shishir made to have a single script callable from either pwsh or bash for the two platforms